client/src/backoffice/Main.tsx

```diff
@@ -152,9 +151,9 @@ const NavigationMenu = withRouter(({ history }) => {
 
 const Landing = () => {
   // TODO: Get user role (Backend WIP)
-  const userRole = 'super';
+  const userRole = 'administrator';
 
-  if (userRole === ROLES.super) return <div>Choose an option from the right side menu</div>;
+  if (userRole === ROLES.administrator) return <div>Choose an option from the right side menu</div>;
```

Comment:
We show this message only for the `administrator`? I guess it's for all type of users so we can safely remove the `if` condition.